### PR TITLE
Hbase

### PR DESCRIPTION
**Pull request by hthayko**

### CODE REVIEW

File: api/services/PeersService.js [hunk 1]
```diff
@@ -1,31 +1,33 @@
 import http from 'superagent'
+var HbaseClient = require('crawler-hbase').Client;
 
 module.exports = {
 
-  listPeers: function() {
-
+  listPeers: function() {
```
Comment (wilsonianb) [inline api/services/PeersService.js]: white space

Comment (hthayko) [inline api/services/PeersService.js]: which one ?

Comment (wilsonianb) [inline api/services/PeersService.js]: at the end of `listPeers: function() {`

Comment (hthayko) [inline api/services/PeersService.js]: should I add a whitespace ?

File: api/services/PeersService.js [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
+          return {
+            'version': r.version,
+            'uptime': parseInt(r.uptime),
+            'inbound_connections': parseInt(r.in_count),
+            'outbound_connections': parseInt(r.out_count),
+            'public_key': r.pubkey,
+            'in_add_count': r.in_add_count,
+            'out_add_count': r.out_add_count,
+            'in_drop_count': r.in_drop_count,
+            'out_drop_count': r.out_drop_count,
```
Comment (wilsonianb) [inline api/services/PeersService.js]: The add/drop counts should have `parseInt`

File: api/services/GraphService.js [hunk 3]
```diff
@@ -1,20 +1,57 @@
...
+            var node = {
+              'version': r.version,
+              'uptime': parseInt(r.uptime),
+              'in': parseInt(r.in_count),
+              'out': parseInt(r.out_count),
+              'public_key': r.pubkey,
+              'in_add_count': r.in_add_count,
+              'out_add_count': r.out_add_count,
+              'in_drop_count': r.in_drop_count,
+              'out_drop_count': r.out_drop_count,
```
Comment (wilsonianb) [inline api/services/GraphService.js]: The add/drop counts should have `parseInt`

File: package.json [hunk 4]
```diff
@@ -11,6 +11,7 @@
     "babel": "^5.8.21",
     "babel-runtime": "^5.8.20",
     "browserify": "^11.0.1",
+    "crawler-hbase": "latest",
```
Comment (stevenzeiler) [inline package.json]: can we please use a hard-coded version number here?

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by stevenzeiler** [comment]
:+1:

**Comment by hthayko** [comment]
first time ever I am getting :+1: from the entire team ! what a day! what a lovely day !

**Pull request merged**